htdocs/includes/db_lib.php

```diff
@@ -6665,7 +6665,7 @@ function search_patients_by_id_count($q, $labsection = 0)
 			"WHERE surr_id LIKE '$q'";
 		} else {
 			$query_string = "SELECT COUNT(DISTINCT patient.patient_id) AS val FROM patient, specimen ".
-					"WHERE patient.surr_id LIKE '$q' AND patient.patient_id = specimen.patient_id ".
```

Comment:
This looks correct in a general sense, but what count will this provide if the user is a lab tech who should be able to see _all_ the patients in a lab? (and their satellite lab ID is null?)